docs/podman.1.md

```diff
@@ -94,8 +94,37 @@ has the capability to debug pods/images created by crio.
 | [podman-version(1)](podman-version.1.md)  | Display the Podman version information.                                        |
 | [podman-wait(1)](podman-wait.1.md)        | Wait on one or more containers to stop and print their exit codes.             |
 
+## FILES
+
+
+**libpod.conf** (`/etc/containers/libpod.conf`)
+
+	libpod.conf is the configuration file for all tools using libpod to manage containers
+
+**storage.conf** (`/etc/containers/storage.conf`)
+
+	storage.conf is the storage configuration file for all tools using containers/storage
+
+	The STORAGE configuration file specifies all of the available container storage options for tools using shared container storage, but in a TOML format that can be more easily modified and versioned.
+
+**mounts.conf** (`/usr/share/containers/mounts.conf` and optionally `/etc/containers/mounts.conf`)
+
+	The mounts.conf files specify volume mount directories that are automatically mounted inside containers when executing the `buildah run` or `buildah build-using-dockerfile` commands.  Container process can then use this content.  The volume mount content does not get committed to the final image.
```

Comment:
Suggest replacing the `buildah` parts with "when running containers with `podman`

Reply:
The "does not get committed" part can go away

Reply:
Nit: "Container process" -> "Container processes"

Reply:
oops correct
podman commit would not commit it though.

Reply:
Fixed buildah references
Fixed Container processes.